policyengine_us/parameters/gov/states/az/tax/income/exemptions/amount/min_age.yaml

```
@@ -0,0 +1,15 @@
+description: Arizona extends the aged exemption to filers at or above this age threshold. 
```

and elsewhere where similar
```suggestion
description: Arizona provides an aged exemption to filers this age or older.
```